packaging/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# This Dockerfile is based on the recommendations provided in the
+# Centos official repository (https://hub.docker.com/_/centos/).
+# It enables systemd to be operational.
+FROM centos:7
+ENV container docker
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == \
+     systemd-tmpfiles-setup.service ] || rm -f $i; done); \
+    rm -f /lib/systemd/system/multi-user.target.wants/*;\
+    rm -f /etc/systemd/system/*.wants/*;\
+    rm -f /lib/systemd/system/local-fs.target.wants/*; \
+    rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
+    rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
+    rm -f /lib/systemd/system/basic.target.wants/*;\
+    rm -f /lib/systemd/system/anaconda.target.wants/*;
+
+RUN yum -y install iproute NetworkManager epel-release cairo* && \
+    yum -y install python2-pip gcc python-devel gobject-introspection-devel && \
+    yum clean all && \
+    pip install -U \
+      pip \
+      setuptools \
+      pbr \
+      tox \
+      \
+      pluggy==0.6 \
+      PyGObject && \
```

Comment:
Do we need to install this via pip for CentOS? Is there nothing packaged to be used? Also I suppose we do not need pip, setuptools, pbr and tox in the distributed image but only for CI, do we?

Reply:
Right, need to remove it.
Not sure about PyGObject, perhaps pygobject2 is enough. I think I had problems with this.
Will check.

Reply:
I was unable to replace these, leaving it to the rpm packaging effort.

Reply:
The PR https://github.com/nmstate/nmstate/pull/162 contains removed all the `pip` command.

Reply:
@cathay4t , lets take it and then start sharing these two images. The dev one should depend on this one (assuming we change this to rpm/s as well).